x-pack/plugins/security_solution/public/common/components/top_n/top_n.tsx

```diff
@@ -95,6 +95,10 @@ const TopNComponent: React.FC<Props> = ({
   const [view, setView] = useState<EventType>(defaultView);
   const onViewSelected = useCallback((value: string) => setView(value as EventType), [setView]);
 
+  useEffect(() => {
```

Comment:
thanks for adding this to handle the scenario where a user changes the Timeline events filter while the graph is already open! 🙏